src/ucp/wireup/select.c

```diff
@@ -1926,6 +1975,11 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
         }
     }
 
+    ucs_memory_type_for_each(mem_type) {
+        ucp_wireup_dev_usage_count_init(&dev_count[mem_type]);
```

Comment:
1. maybe it would be nicer to pass excl_lane as a parameter to ucp_wireup_dev_usage_count_init
2. I think we need to do it before line 2006, inside the loop that goes over rndv_modes

Reply:
Done.